kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/Kroxylicious.java

```diff
@@ -39,16 +39,42 @@ public Kroxylicious(String deploymentNamespace) {
         this.containerImage = kroxyUrl + Environment.KROXY_VERSION;
     }
 
+    private void deployDefaultConfigMap(String clusterName) {
```

Comment:
naming.. in kube parlance, things like configmaps are 'created' rather than 'deployed'.

Reply:
done